Makefile

```diff
@@ -1471,7 +1473,7 @@ ifeq ($(PLATFORM), OS_SOLARIS)
 	ROCKSDBJNILIB = librocksdbjni-solaris$(ARCH).so
 	ROCKSDB_JAR = rocksdbjni-$(ROCKSDB_MAJOR).$(ROCKSDB_MINOR).$(ROCKSDB_PATCH)-solaris$(ARCH).jar
 	JAVA_INCLUDE = -I$(JAVA_HOME)/include/ -I$(JAVA_HOME)/include/solaris
-	SHA256_CMD = digest -a sha256
+	SHA1_CMD = digest -a sha1
```

Comment:
@sagar0 Okay cool. I just rebased to remove the merge conflict, I think assuming Travis works, then we should be good to go now.